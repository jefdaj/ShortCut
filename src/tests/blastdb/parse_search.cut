result = parse_searches ["Synechococcus elongatus PCC 7942 (refseq)"]
