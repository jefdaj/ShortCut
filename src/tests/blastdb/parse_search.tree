.
|-- cache
|   `-- list
|       `-- dd81c4f27d.txt
|-- exprs
|   |-- list
|   |   `-- df2095ac89.str.list -> ../../cache/list/dd81c4f27d.txt
|   |-- parse_searches
|   |   `-- 42a03b76c5.search
|   `-- str
|       `-- 45914ff33c.str
`-- vars
    `-- result -> ../exprs/parse_searches/42a03b76c5.search

7 directories, 5 files
