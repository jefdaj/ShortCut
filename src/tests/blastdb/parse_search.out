organism	database	identifier
Synechococcus elongatus PCC 7942	refseq	NA

